DEVELOPING.md

```diff
@@ -85,42 +85,6 @@ If all goes well, you'll see some messages about things being all good, and
 your services should behave normally. If they don't, exit the studio and
 re-enter.
 
-If you try to run a build and get an error message in the logs about
```

Comment:
Also should add instruction here on how to do the migration for a dev env...

Reply:
Also can you please open an issue to update key docs - eg, the 'on-call' docs which deal with the DB instructions, and follow up on that once this merges.

Reply:
https://github.com/habitat-sh/builder/issues/620